.github/workflows/pull-request.yml

```diff
@@ -8,10 +8,10 @@ on:
 
 jobs:
     macos-build:
-        runs-on: macos-latest
+        runs-on: macOS-13
```

Comment:
Whats the reason for this change? `latest` would seem like a better choice?

Reply:
It does @dabeeeenster, this seems to have come with the original PR. I'll switch it back and check that everything builds...

Reply:
Hi @dabeeeenster if you check out this file again I've added a load of comments that should be self-explanatory. Turns out latest is like 3/4 years old, so specifying the version actually gets us newer xCode / MacOS.

I've added 13 and 14 to cover us for now. 12 (latest) didn't have the build tools we need.

Reply:
Huh weird. OK yep hard coding sounds like the only path.